test/JDBC/input/BABEL-4231-vu-verify.mix

```diff
@@ -157,3 +157,7 @@ GO
 -- psql
```

Comment:
please add test cases reported to be failing from jira.

Reply:
Addressed.